go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/argoproj/argo-rollouts v1.4.1
 	github.com/cenkalti/backoff/v4 v4.2.1
 	github.com/cloudevents/sdk-go/v2 v2.14.0
-	github.com/dapr/components-contrib v1.13.0-rc.2.0.20240503231149-1f46231d875c
+	github.com/dapr/components-contrib v1.13.1
```

Comment:
Revert?

Reply:
The problem was the azure sdk version, but it's already been fixed in components-contrib master, so we can point to the latest commit, for now, until we tag a version, if that's ok.